src/LibChorus/VcsDrivers/Mercurial/HgHighLevel.cs

```diff
@@ -6,9 +6,19 @@ namespace Chorus.VcsDrivers.Mercurial
 {
 	public class HgHighLevel
 	{
+		public static string MakeCloneFromUsbToLocal(string sourcePath, string targetDirectory, IProgress progress)
+		{
+			return MakeCloneFromLocalToLocal(sourcePath, targetDirectory, true, progress);
+		}
+
+		public static string MakeCloneFromLocalToUsb(string sourcePath, string targetDirectory, IProgress progress)
+		{
+			return MakeCloneFromLocalToLocal(sourcePath, targetDirectory, false, progress);
+		}
+
 		//TODO: get rid of this, or somehow combine it with the other Clone() options out there
 		/// <returns>path to clone, or empty if it failed</returns>
-		public static string MakeCloneFromLocalToLocal(string sourcePath, string targetDirectory, bool alsoDoCheckout, IProgress progress)
+		private static string MakeCloneFromLocalToLocal(string sourcePath, string targetDirectory, bool alsoDoCheckout, IProgress progress)
```

Comment:
This variable now affects another behavior as well, might be good to rename it, maybe something like fromUsb?